exposure.py

```diff
@@ -0,0 +1,62 @@
+import math
+import csv
+from collections import Counter
+
+def parse_exposure_file(file_location):
+    """
+    :param file: File location
+    :return: Dictionary of exposures
+    { <name>: { min: <min>, max: <max> } }
+    """
+    exposures = []
+    with open(file_location, 'r') as filename:
+        reader = csv.DictReader(filename)
+        for row in reader:
+            if 'name' not in row or \
+                'min' not in row or \
+                'max' not in row:
+                raise Exception('''
+                    You must provide a min, max and name
+                    for each row - got {}.
+                    '''.format(row)
+                )
+            exposures.append({
+                'name': row['name'],
+                'min': float(row['min']),
+                'max': float(row['max']),
+            })
+
+    return exposures
+
+
+def get_exposure_args(existing_rosters, exposure_bounds):
```

Comment:
There is probably an actual optimizer-based way to do this where the min and max for each player are constraints. The question is what you would optimize on - maybe aggregate score? This would be a fairly large-scale refactor so I'm okay starting with this and providing some override args (ex. custom function to prevent certain player combos (ex. three highly priced players))l.